src/components/FollowButton/index.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import './style.sass';
+import { isLoggedIn, api, getToken } from '../../utils';
+
+export class FollowBtn extends Component {
```

Comment:
the naming here should be precise, short form is not ideal, you better have a long name but readable.